src/ResourceManagement/Network/Microsoft.Azure.Management.Network/Properties/AssemblyInfo.cs

```diff
@@ -19,8 +19,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Network Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Network management functions for managing the Microsoft Azure Network service.")]
 
-[assembly: AssemblyVersion("7.1.2.0")]
-[assembly: AssemblyFileVersion("7.1.2.0")]
+[assembly: AssemblyVersion("8.0.0.0")]
```

Comment:
ApplicationGatewayBackendHealthServer object is used in one GET command only. This is output only object and customer doesn't used it for any input. 
Impact of change would be that customer would get health output as healthy/unhealthy (as string) instead of up/down.